templates/templates/navigation-canonical-h.html

```diff
@@ -0,0 +1,193 @@
+          </div>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="p-media-object">
+          <img class="p-media-object__image u-hide--small" src="data:image/svg+xml,%3C%3Fxml%20version%3D%271.0%27%20encoding%3D%27UTF-8%27%20standalone%3D%27no%27%3F%3E%0A%3Csvg%20xmlns%3D%27http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%27%20xml%3Aspace%3D%27preserve%27%20height%3D%27400px%27%20width%3D%27400px%27%20version%3D%271.1%27%20enable-background%3D%27new%20-97%20220.9%20400%20400%27%20viewBox%3D%27-97%20220.9%20400%20400%27%3E%0A%3Cstyle%20type%3D%27text%2Fcss%27%3E%0A%09.st0%7Bfill%3A%23E95420%3B%7D%0A%09.st1%7Bfill%3A%23FFFFFF%3B%7D%0A%3C%2Fstyle%3E%0A%09%3Cpath%20class%3D%27st0%27%20d%3D%27m303%20420.9c0%20110.4-89.5%20200-200%20200s-200-89.6-200-200c0-110.5%2089.5-200%20200-200s200%2089.5%20200%20200z%27%2F%3E%0A%09%09%3Cpath%20class%3D%27st1%27%20d%3D%27m-33%20394.2c-14.8%200-26.7%2012-26.7%2026.7s12%2026.7%2026.7%2026.7%2026.7-12%2026.7-26.7c0-14.8-12-26.7-26.7-26.7zm190.7%20121.3c-12.8%207.4-17.1%2023.7-9.8%2036.5%207.4%2012.8%2023.7%2017.2%2036.5%209.8s17.1-23.7%209.8-36.5c-7.5-12.7-23.8-17.1-36.5-9.8zm-132.7-94.6c0-26.4%2013.1-49.7%2033.2-63.8l-19.5-32.7c-23.4%2015.6-40.7%2039.5-48%2067.4%208.4%206.9%2013.8%2017.4%2013.8%2029.1s-5.4%2022.2-13.8%2029.1c7.2%2027.9%2024.6%2051.8%2048%2067.4l19.5-32.7c-20.1-14.1-33.2-37.4-33.2-63.8zm78-78c40.8%200%2074.2%2031.2%2077.7%2071.1l38.1-0.6c-1.9-29.4-14.7-55.8-34.5-75.2-10.2%203.8-21.9%203.2-32-2.6-10.2-5.9-16.5-15.7-18.3-26.5-9.9-2.7-20.3-4.2-31-4.2-18.5%200-35.9%204.3-51.4%2012l18.6%2033.2c9.9-4.6%2021.1-7.2%2032.8-7.2zm0%20156c-11.7%200-22.9-2.6-32.9-7.2l-18.6%2033.2c15.5%207.7%2033%2012%2051.4%2012%2010.7%200%2021.1-1.5%2031-4.2%201.7-10.7%208.1-20.6%2018.3-26.5%2010.1-5.9%2021.9-6.4%2032-2.6%2019.7-19.4%2032.6-45.8%2034.5-75.2l-38.1-0.6c-3.4%2039.8-36.8%2071.1-77.6%2071.1zm54.6-172.7c12.8%207.4%2029.1%203%2036.5-9.8s3-29.1-9.8-36.5-29.1-3-36.5%209.8c-7.3%2012.8-2.9%2029.1%209.8%2036.5z%27%2F%3E%0A%3C%2Fsvg%3E" alt="">
+          <div class="p-media-object__details">
+            <h3 class="p-media-object__title"><a href="https://ubuntu.com/support">Ubuntu Advantage&nbsp;&rsaquo;</a></h3>
+            <p class="p-media-object__content u-hide--small">Extended Security Maintenance, Kernel Livepatch, FIPS, enterprise support and certification.</p>
+            <p class="p-p--small u-no-margin--bottom u-hide--small"><a href="http://ubuntu.com/advantage">Your subscriptions&nbsp;&rsaquo;</a></p>
+            <p class="p-p--small u-no-margin--bottom u-hide--small"><a href="https://support.canonical.com/">Support login&nbsp;&rsaquo;</a></p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="dropdown-window__content">
+  <div class="u-fixed-width"><hr></div>
+  <div class="row">
+    <div class="col-9" style="margin-bottom:24px;">
```

Comment:
@anthonydillon is right, we should refrain from using both inline styles and pixel values wherever possible.

```suggestion
    <div class="col-9 u-sv3">
```